src/tests/components/Alert/index.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+import { render } from '@testing-library/react'
+
+import bellIcon from 'assets/images/icons/bell.svg'
+import errorIcon from 'assets/images/icons/error.svg'
+import Alert, { alertIcon } from 'components/Alert'
+
+describe('given Alert component is mounted', () => {
+  it('renders correct title', () => {
+    const element = render(<Alert title="someTitle" detail="someDetail" icon={alertIcon.bell} />)
```

Comment:
How about destructuring?
```suggestion
    const { getByText } = render(<Alert title="someTitle" detail="someDetail" icon={alertIcon.bell} />)
```

Reply:
Fixed in f788bf40fa566a2b845815d0da30e6a88704ff2d